html/Corporate/index.html

```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Corporate</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <!-- Banner -->
+  <section class="banner">
+    <div class="banner__container">
+      <!-- Title -->
+      <header class="banner__title">
+        There is no other platforms for you as like 
+      </header>
+
+      <!-- Play -->
+      <button type="button" class="banner__play">
+        <img  src="./assets/play-icon/play-video-icon.svg" alt="">
+      </button> 
+
+      <!-- Button -->
+      <button class="banner__try-now" type="button">Try now</button>
+
+      <!-- Description -->
+      <p class="banner__description">* No need to add cards details</p>
+    </div>
+  </section>
+
+  <!-- Unique platform -->
+  <section class="unique-platform">
+      
+    <!-- Left side -->
+    <div class="unique-platform__left-side">
+    
+      <!-- Type -->
+      <p class="unique-platform__type">New design</p>
+
+      <!-- Title -->
+      <header class="unique-platform__title section-title">
+        There is no other platforms for you as like <span class="section-title__dots">....</span>
+      </header>
+     
+      <!-- Description -->
+      <p class="unique-platform__description unique-platform__description--margin">
```

Comment:
also avoid to use css property names in classes, I would suggest "spaced", "extra space". As in the case with content inside class name this approach would confuse if the type of CSS property was changed but the result has remained the same